test/integ_tests/test_device_creation.py

```diff
@@ -77,7 +73,7 @@ def test_get_devices_others():
 
 def test_get_devices_all(braket_devices):
     result_arns = [result.arn for result in braket_devices]
-    for arn in [RIGETTI_ARN, IONQ_ARN, SIMULATOR_ARN, OQC_ARN]:
+    for arn in [RIGETTI_ARN, IONQ_ARN, SIMULATOR_ARN]:
```

Comment:
Why haven't we added IQM_ARN here as well?

Reply:
Sure, I'm fine with that